kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx

```diff
@@ -52,6 +52,7 @@ const Overview: React.FC<Props> = ({
             label="URP"
             title="Under replicated partitions"
             isAlert
+            alertType={underReplicatedPartitions === 0 ? 'error' : 'success'}
```

Comment:
pls cover your code with tests

Reply:
Actually we need to add specs for this particular line of code& https://sonarcloud.io/component_measures?id=provectus_kafka-ui_frontend&metric=new_coverage&pullRequest=1336&selected=provectus_kafka-ui_frontend%3Asrc%2Fcomponents%2FTopics%2FTopic%2FDetails%2FOverview%2FOverview.tsx&view=list